src/components/ui/use-toast.ts

```diff
@@ -0,0 +1,194 @@
+'use client';
+
+// Inspired by react-hot-toast library
+import * as React from 'react';
```

Comment:
I know it's very cumbersome to fix, but could you refactor this wildcard import?